### ISSUE TITLE
CI failure tracker

### ISSUE THREAD

**Issue by Tcc0403** [linkedin/Liger-Kernel#729]
### 🐛 Describe the bug

Most failures are related to transformers VLM changes

### qwen2vl_mrope
- [ ] test_qwen2vl_mrope https://github.com/linkedin/Liger-Kernel/pull/728
### monkey patch
- [ ] test_monkey_patch::test_apply_liger_kernel_to_instance_for_mllama_for_conditional_generation
- [ ] test_monkey_patch::test_apply_liger_kernel_to_instance_for_gemma3 
- [ ] test_monkey_patch::test_apply_liger_kernel_to_instance_for_qwen2_vl
- [ ] test_monkey_patch::test_apply_liger_kernel_to_instance_for_qwen2_5_vl

### VLM changes related (config, video_processor, ...)

- [ ] llava
- [ ] geamm3
- [ ] qwen2_vl
- [ ] qwen2_5_vl

### _init_weights related

- [ ] olmo2
- [ ] glm4
- [ ] gemma1
- [ ] gemma2

### logits numerical issue
- [ ] gemma3 

### Versions

liger_kernel==https://github.com/linkedin/Liger-Kernel/commit/5d25e46827cd63210373d7a15641438b0bf36c66
trasnformers==4.52.3

**Comment by lancerts** [comment]
@momochen for visability.

**Comment by shimizust** [comment]
@Tcc0403 Thanks for doing the initial investigation. Can I help look into some of these? Which failures are you currently looking at?

**Comment by Tcc0403** [comment]
@shimizust Thanks for the help. I only checked the vlm related issues, haven't looked at what caused the `_init_weights` errors.

**Comment by vaibhavjindal** [comment]
#741 should fix all the _init_weights related errors
